pkg/apis/numaplane/v1alpha1/isbservicerollout_types.go

```diff
@@ -26,7 +26,12 @@ import (
 
 // ISBServiceRolloutSpec defines the desired state of ISBServiceRollout
 type ISBServiceRolloutSpec struct {
-	InterStepBufferService runtime.RawExtension `json:"interStepBufferService"`
+	InterStepBufferService InterStepBufferService `json:"InterStepBufferService"`
```

Comment:
```suggestion
	InterStepBufferService InterStepBufferService `json:"interStepBufferService"`
```

Reply:
shouldn't this be lower case?

Reply:
Yes, it should match the numaflow version of it.

Reply:
https://github.com/numaproj/numaflow/blob/main/pkg/apis/numaflow/v1alpha1/pipeline_types.go#L453

Reply:
taking back my response, this one doesn't need to match. The service name one needs to.